packages/runtime/container-runtime/src/summarizerHeuristics.ts

```diff
@@ -67,20 +67,44 @@ export class SummarizeHeuristicRunner implements ISummarizeHeuristicRunner {
         return this.heuristicData.lastOpSequenceNumber - this.heuristicData.lastSuccessfulSummary.refSequenceNumber;
     }
 
-    public run() {
+    public run(numSystemOps?: number, numNonSystemOps?: number) {
```

Comment:
Optionality of arguments suggests we are not doing right thing for some path (potentially first summary). Can you please take a look?